frontend/views/pages/GroupDashboard.vue

```diff
@@ -2,19 +2,23 @@
 page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
   template(#title='') {{ groupSettings.groupName }}
 
+  welcome(v-if='groupIncomeDistribution.length === 0')
```

Comment:
I think there should only be a single card ever displayed, so `welcome` and `start-inviting` should be combined somehow. I think @mmbotelho also talks about this [in her comment here](https://github.com/okTurtles/group-income-simple/pull/789/files#r365950977).

Reply:
Okay, I'll wait for @pieer PR to be merged and then I'll update it with this file